vignettes/opart_gaussian.Rmd

```diff
@@ -187,7 +187,7 @@ for(profile_id in profile_ids_vec){
 ```
 
 
-In the above code we use 50 profile ids from neuroblastoma dataset and run the 3 algorithms on the logratio values. We also maintain "iters" variable to count how many times we successfully run each iteration. In the "if" condition at the bottom of the loop we check if the segment ends differ and break from the loop. Therefore, if all the segment ends of opart matches with fpop and cpt.mean then "iters" should be 10 after exiting the for loop.
+In the above code we use 10 profile ids from neuroblastoma dataset and run the 3 algorithms on the logratio values. We also maintain "iters" variable to count how many times we successfully run each iteration. In the "if" condition at the bottom of the loop we check if the segment ends differ and break from the loop. Therefore, if all the segment ends of opart matches with fpop and cpt.mean then "iters" should be 10 after exiting the for loop.
```

Comment:
please use a variable like `n.profiles <- 10` and use inline code to insert that value in the text here https://rmarkdown.rstudio.com/lesson-4.html